src/python/WMCore/Services/LogDB/LogDBBackend.py

```diff
@@ -86,13 +96,35 @@ def prefix(self, mtype):
             mtype = 'agent-%s' % mtype
         return mtype
 
-    def post(self, request, msg='', mtype="comment"):
-        """Post new entry into LogDB for given request"""
+    def agent_update(self, request, msg='', mtype="comment"):
```

Comment:
default value shouldn't be comment, maybe info, or we don't have to have default value.
``` python
agent_update(self, request, msg='', mtype="info"):
```